Set up the Debian-package-specific path for documentation.

This patch by Kęstutis Biliūnas.
This patch will not be submitted upstream.

--- a/fontforge/prefs.c
+++ b/fontforge/prefs.c
@@ -984,9 +984,9 @@
 #if defined(__MINGW32__)
 	helpdir = copy("");
 #elif defined(DOCDIR)
-	helpdir = copy(DOCDIR "/");
+	helpdir = copy(DOCDIR "/html/");
 #elif defined(SHAREDIR)
-	helpdir = copy(SHAREDIR "/doc/fontforge/");
+	helpdir = copy(SHAREDIR "/doc/fontforge/html/");
 #else
 	helpdir = copy("/usr/local/share/doc/fontforge/");
 #endif
--- a/fontforge/uiutil.c
+++ b/fontforge/uiutil.c
@@ -316,9 +316,9 @@
 	if ( ! GFileIsAbsolute(file) ) {
 	    if ( helpdir==NULL || *helpdir=='\0' ) {
 #ifdef DOCDIR
-		strcpy(fullspec,DOCDIR "/");
+		strcpy(fullspec,DOCDIR "/html/");
 #elif defined(SHAREDIR)
-		strcpy(fullspec,SHAREDIR "/doc/fontforge/");
+		strcpy(fullspec,SHAREDIR "/doc/fontforge/html/");
 #else
 		strcpy(fullspec,"/usr/local/share/doc/fontforge/");
 #endif
@@ -329,7 +329,7 @@
 	if (( pt = strrchr(fullspec,'#') )!=NULL ) *pt ='\0';
 	if ( !GFileReadable( fullspec )) {
 	    if ( *file!='/' ) {
-		strcpy(fullspec,"/usr/share/doc/fontforge/");
+		strcpy(fullspec,"/usr/share/doc/fontforge/html/");
 		strcat(fullspec,file);
 		if (( pt = strrchr(fullspec,'#') )!=NULL ) *pt ='\0';
 	    }
